### DISCUSSION TITLE
Changed date as a parameter

### DISCUSSION THREAD

**Discussion by RoyalWulf** [nbrown02/FlowViz#131] [Ideas]
Great power bi, thank you. To make life easier for us at least, can the code be changed so that a Changed date form bring in data be a parameter? We have lots of workitems and the current default dates makes refreshing data very slow. I can change the dates but they appear in several queries.  It would be easier to have this as a parameter.  Cheers.

**Comment by nbrown02** [comment]
Hi there,

Thanks - this isn't something I am looking to do currently, mainly due to the tables being of different types. For example the "WorkItems" table contains a row for each item and its current status where as "WorkItemRevisions" contains all revisions to a work item, which is far greater in size. To apply a date parameter across the board wouldn't work given this setup. You are welcome to add your own though!

**Reply by RoyalWulf** [reply]
Hi, thanks for your reply.
It sounds like i need different dates for different tables so i need to be careful setting my own?
Is there any rules i should follow?
I spent aver a day getting an initial load, it kept timing out, we have a lot of data.
Cheers

**Reply by nbrown02** [reply]
If you take the general guidance here - https://github.com/nbrown02/FlowViz/discussions/62

The 6 month tables are the big ones so be careful. If ever i have data issues, limiting it to the last 3 (and/or filtering by team) fixes it :)
